examples/multiforwarder/README.md

```diff
@@ -1,42 +0,0 @@
-## Requires
```

Comment:
Can we just disable this example?

Reply:
@pperiyasamy How are you on getting the cmd-forwarder-ovs into a place we could put it in the examples?

Reply:
@denis-tingaikin One option is to simply use the cmd-forwarder-vpp and not provide it a SRIOV config (and thus it won't handle SRIOV case).

Reply:
@edwarnicke The cmd-forwarder-ovs runs only in [kernel datapath](https://github.com/networkservicemesh/cmd-forwarder-ovs/blob/main/main.go#L227-L231) if no SRIOV config provided. when sriov config exists, then it supports both kernel and smart nic, the connection type would be chosen based on client service request URL.